students/douglas_klos/session7/html_render.py

```diff
@@ -1,18 +1,163 @@
 #!/usr/bin/env python3
 
-"""
-A class-based system for rendering html.
-"""
+""" A class based system for rendering HTML """
 
+# Douglas Klos
+# February 27th, 2019
+# Python 210, Session 7
+# html_render.py
 
-# This is the framework for the base class
-class Element(object):
 
-    def __init__(self, content=None):
-        pass
+class Element():
+    """ Framework for a basic element in HTML code """
+    tag = 'html'
+    indent = ''
+
+    def __init__(self, content=None, **kwargs):
+        self.attributes = kwargs
+
+        if content is not None:
```

Comment:
_None_ always returns False when you test for truth, so you can simplify this line:

```
if not content:
```

A few more information on which values test False in Python (an older document, but still holds):

https://docs.python.org/2.4/lib/truth.html